src/android/InAppBrowser.java

```diff
@@ -271,7 +272,7 @@ public void run() {
                     } else {
                         ((InAppBrowserClient)inAppWebView.getWebViewClient()).waitForBeforeload = false;
                     }
-                    inAppWebView.loadUrl(url);
+                    inAppWebView.loadUrl(url, null);
```

Comment:
Why all those changes to the `loadUrl` calls if we don't actually change the method signature ourselves? Couldn't we just keep these as they are and only add the `headers` in the places where we need it?

Reply:
I guess this was to explicitely mark headers were null for easier code navigation